acceptance/tests/utf8/utf8-in-catalog.rb

```diff
@@ -0,0 +1,100 @@
+test_name 'utf-8 characters in cached catalog' do
+  utf8chars = "€‰ㄘ万竹ÜÖ"
+  file_content = "This is the file content. file #{utf8chars}"
+  tmpdir = master.tmpdir("code")
+  on(master, "rm -rf #{tmpdir}")
+  env_dir = "#{tmpdir}/environments"
+  agents.each do |agent|
+    agent_file = agent.tmpfile("file" + utf8chars) 
+
+    step "Apply manifest" do
+      on(agent, "rm -rf #{agent_file}")
+    
+      master_manifest =
+<<PP
```

Comment:
this is hard to read and makes the indenting less valuable.  the heredoc should be up next to the ` = `